src/Quota_Command.php

```diff
@@ -367,19 +391,21 @@ public function add( $args, $assoc_args ) {
 	 * [<id>]
 	 * : The id of the site to set the quota.
 	 *
-	 * [<quota-to-subtract-in-mb>]
-	 * : Subtract quota value in mb
+	 * [<quota-to-subtract>]
+	 * : Subtract quota value in mb or gb
 	 *
 	 *
 	 * ## EXAMPLES
 	 *
 	 *     # Decreasing the quota of blog 2
-	 *     $ wp quota substract 2 2500
-	 *     $ wp quota substract 2500 --url=subsite.local
+	 *     $ wp quota subtract 2 2500
+     *     $ wp quota subtract 2500 --url=subsite.local
+     *     $ wp quota subtract 5g --url=subsite.local
+     *          # Decreases quota by 5gb
 	 *
 	 *     Quota is now 10000 MB for subsite.local
 	 *
-	 * @subcommand substract
+	 * @subcommand subtract
 	 */
 	public function substract( $args, $assoc_args ) {
```

Comment:
Please use **subtract** consequently within this method ;-) No mix of subtract and substract.